src/parser/lessParser.ts

```diff
@@ -361,9 +488,10 @@ export class LESSParser extends cssParser.Parser {
 	}
 
 	public _parseInterpolation(): nodes.Node {
-		//  @{name}
+		// @{name} Variable or 
+		// ${name} Property
```

Comment:
Same as the other `${prop}` confusion. Can you point me to the docs that describe this feature?

Reply:
The documentation is very brief and doesn't cover the interpolated form. It's here: http://lesscss.org/features/#variables-feature-properties-as-variables-new-

It should probably be added to Less docs, but it's valid. In a lot of cases for these, I pulled code directly from Less unit tests and added them to the language service tests. In fact, I opened Less unit tests in VSCode to see what was failing first, before adding language service tests.